hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/GetScmRatisRolesSubcommand.java

```diff
@@ -50,13 +52,54 @@ public class GetScmRatisRolesSubcommand extends ScmSubcommand {
       description = "Format output as JSON")
   private boolean json;
 
+  @CommandLine.Option(names = { "--table" },
+       defaultValue = "false",
+       description = "Format output as Table")
+  private boolean table;
+
+  private static final String SCM_ROLES_TITLE = "Storage Container Manager Roles";
+
+  private static final List<String> RATIS_SCM_ROLES_HEADER = Arrays.asList(
+      "Host Name", "Ratis Port", "Node ID", "Role", "Host Address");
+
+  private static final List<String> STANDALONE_SCM_ROLES_HEADER = Arrays.asList("Host Name", "Port");
+
   @Override
   protected void execute(ScmClient scmClient) throws IOException {
     List<String> ratisRoles = scmClient.getScmRatisRoles();
+    boolean isRatisEnabled = scmClient.isScmRatisEnable();
     if (json) {
       Map<String, Map<String, String>> scmRoles = parseScmRoles(ratisRoles);
       System.out.print(
           JsonUtils.toJsonStringWithDefaultPrettyPrinter(scmRoles));
+    } else if (table) {
+      FormattingCLIUtils formattingCLIUtils = new FormattingCLIUtils(SCM_ROLES_TITLE);
+
+      // Determine which header to use based on whether Ratis is enabled or not.
+      if (isRatisEnabled) {
+        formattingCLIUtils.addHeaders(RATIS_SCM_ROLES_HEADER);
+      } else {
+        formattingCLIUtils.addHeaders(STANDALONE_SCM_ROLES_HEADER);
+      }
+
+      for (String role : ratisRoles) {
+        String[] roleItems = role.split(":");
+        if (roleItems.length < 2) {
+          err.println("Invalid response received for ScmRatisRoles.");
+        }
+
+        // In STANDALONE mode, output only the Host Name and Port information.
+        if (roleItems.length == 2) {
+          formattingCLIUtils.addLine(roleItems[0], roleItems[1]);
```

Comment:
Can we pass `roleItems` directly?

```suggestion
          formattingCLIUtils.addLine(roleItems);
```